staging/src/k8s.io/client-go/testing/fixture_test.go

```diff
@@ -63,6 +63,9 @@ func TestWatchCallNonNamespace(t *testing.T) {
 	codecs := serializer.NewCodecFactory(scheme)
 	o := NewObjectTracker(scheme, codecs.UniversalDecoder())
 	watch, err := o.Watch(testResource, ns)
+	if err != nil {
+		t.Errorf("test resource watch failed in %s: %v ", ns, err)
```

Comment:
Isn't this fatal for the test?

Reply:
Same below.

Reply:
It seems that the test is not trigger correctly.

Reply:
but then `watch` is probably nil, isn't it? So we should terminate the test early.

Reply:
In func (t *tracker) Watch, it will call NewFakeWithChanSize and returns fakewatcher as a watch. So watch will not be nil forever. But in func (t *tracker) Watch the return error seems to be nil forever too. This PR seems not necessary.

Reply:
Cannot follow. If err is non-nil, watch will probably be nil and the test panic. This needs a `t.Fatalf`.

Reply:
Done. Oh, you are right, I mistake it. Thanks for your review.